### ISSUE TITLE
Turn off saving of code in db container as I dont want it to use extra space I am already saving it when it returns

### ISSUE THREAD

**Issue by Harsh4999** [judge0/judge0#384]

**Comment by ryankwondev** [comment]
I don't see any logic to stop saving the DB (which would be necessary due to the way judge0 works). If you need that functionality, you could either have a batch job periodically clean the DB, or create a callback that deletes it whenever a submission finishes processing.

**Comment by Harsh4999** [comment]
Yeah I agree but there should be an option to configure that like I understand it would break if someone enables not saving in DB container and also doesn't provide valid db source instead but there should be a way where we can configure DB urls and than it uses that DB instead of inbuilt one same applies to REDIS

**Comment by ryankwondev** [comment]
You can check out that two configs here in `judge0.conf`file:
https://github.com/judge0/judge0/blob/07f78a57013253eae591fb42555c6468c01c7aa1/judge0.conf#L218
https://github.com/judge0/judge0/blob/07f78a57013253eae591fb42555c6468c01c7aa1/judge0.conf#L196
